docs/gettingStarted/ANGULAR.md

```diff
@@ -1 +1,242 @@
+
+```json
+{
+    "plugins": [
+        {
+            "name": "@zeplin/cli-connect-angular-plugin"
+        }
+    ],
+    "projects": [],
+    "styleguides": [
+        "5cd486b18a64c1414be004fb"
+    ],
+    "components": [
+        {
+            "path": "src/components/Button/Button.ts",
+            "zeplinNames": [
+                "Controls / Button / Primary"
+            ]
+        }
+    ]
+}
+```
+
+Next up, we'll run the CLI tool!
+
+## 4. Run Zeplin CLI
+
+It's time! Let's **run the CLI tool and see Connected Components in action** within Zeplin. 🎉
+
+Run the following command—if it's your first time, you'll need to login to Zeplin first:
+
+```sh
+zeplin connect
+```
+
+Now head back to Zeplin and click on one of the components you connected. You should see an output similar to this:
+
+<img src="../../img/zeplinConnectedComponent-angular.png" alt="Connected component in Zeplin" width="600" />
+
+## 5. Add links _(Optional)_
+
+Connected Components also lets you add links to various sources like your Storybook, repository, wiki and so on. In the screenshot above, notice that we have links to GitHub and Storybook.
+
+To add links to your components, check out these guides:
+
+- [Adding Storybook links](../link/STORYBOOK.md)
+- [Adding Styleguidist links](../link/STYLEGUIDIST.md)
+- [Adding repository links](../link/REPOSITORY.md), e.g. GitHub, GitLab, Bitbucket
+- [Adding custom links](../link/CUSTOM.md), e.g. internal Design System wiki
+
+## 6. More code snippets _(Optional)_
+
+By default, Angular plugin generates a code snippet using only one of the possible selectors of the Angular component. In most cases the generated code snippet will be enough to get a glimpse of how it can be use, but if you want to see all possible combinations, you can use following configuration to do so.
+
+```json
+…
+    "plugins" : [{
+        "name": "@zeplin/cli-connect-angular-plugin",
+        "config": {
+            "useFullSnippet": true,
+            "useFullDescription": true,
+        }
+    }],
+…
+```
+
+The output will be something like the below:
+
+<img src="../../img/zeplinConnectedComponent-angular-full.png" alt="Connected component in Zeplin" width="600" />
+
+## 6. Connect more components
+
+Now that we connected our very first component, you can go ahead and connect more! Check out the section [Add component from codebase](#add-component-from-codebase) if you need any help.
+
+Hope this getting started guide was helpful, reach out to us at [support@zeplin.io](mailto:support@zeplin.io) if you have any questions or feedback.
+
+For further details on how to customize the configuration file, check out the [Configuration file documentation](CONFIGURATION_FILE.md).
+
+# Troubleshooting
+
+If you run into any issues while running the `zeplin connect` command, make sure that you have the Angular plugin installed by running the following command:
+
+```sh
+npm list -g --depth 0 @zeplin/cli-connect-angular-plugin
+```
```

Comment:
Added this also in https://github.com/zeplin/connected-components-docs/pull/5/files#diff-40e74671d63b2d7de03e972cc49d61afR214-R216